mirdata/annotations.py

```diff
@@ -38,17 +38,17 @@ class SectionData(Annotation):
     """SectionData object
 
     Attributes:
-        intervals (np.ndarray or None): (n x 2) array of intervals
+        intervals (np.ndarray): (n x 2) array of intervals
             (as floats) in seconds in the form [start_time, end_time]
             times should be positive and intervals should have
             non-negative duration
-        labels (list or None): list of labels (as strings)
+        labels (np.ndarray or None): list of labels (as strings)
```

Comment:
I think this should be list for `mir_eval` no? otherwise change the description as well

Reply:
ah whoops, i messed up the docstring! it should indeed be a list. will update